sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_user_agent.py

```diff
@@ -1,8 +0,0 @@
-# ------------------------------------
-# Copyright (c) Microsoft Corporation.
-# Licensed under the MIT License.
-# ------------------------------------
-import platform
-from ._version import VERSION
-
-USER_AGENT = f"azsdk-python-appconfiguration/{VERSION} Python/{platform.python_version()} ({platform.platform()})"
```

Comment:
Then where do we define the UA string?

Reply:
In azure-core: https://github.com/Azure/azure-sdk-for-python/blob/4ed44df82b3ef68569f819710306797ed78abab4/sdk/core/azure-core/azure/core/pipeline/policies/_universal.py#L219-L221